src/ec/packages/ec-component-social-media-follow/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@ecl-twig/ec-component-social-media-follow",
+  "author": "European Commission",
+  "license": "EUPL-1.1",
+  "version": "0.0.1-alpha",
+  "description": "ECL Twig - EC Social Media Follow",
+  "publishConfig": {
+    "access": "public"
+  },
+  "devDependencies": {
+    "@ecl/ec-resources-social-icons": "~2.1.1"
```

Comment:
even if the data are not directly used, please add the specs to devDependencies (more future-proof)